### PR TITLE
added documentation for ultrasonic sensor / fixes issue w other files

### PR DESCRIPTION
**Pull request by jyanev**

### CODE REVIEW

**Review by SwindleA — APPROVED** [review]
looks good

File: pi4micronaut-utils/src/docs/asciidoc/components/inputComponents/ultraSonicSensor.adoc [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+
+Circuit Diagram:
+
+image::https://docs.sunfounder.com/projects/davinci-kit/en/latest/_images/image329.png[]
+
+===== Functionality
+The Ultrasonic Sensor uses sound waves to calculate the distance from itself to the surface it is pointed at.
+The resulting measurement is accurate within 3mm and has a stable signal within 5m of the sensor,
+gradually weakening until fully it fully disappears at 7m.
```
Comment (ruthvikm) [inline pi4micronaut-utils/src/docs/asciidoc/components/inputComponents/ultraSonicSensor.adoc:30]: Sentence is not clear

Comment (ruthvikm) [reply pi4micronaut-utils/src/docs/asciidoc/components/inputComponents/ultraSonicSensor.adoc:30]: Also include the info about trigger and echo channels on the ultrasonic sensor

File: pi4micronaut-utils/src/docs/asciidoc/components/inputComponents/ultraSonicSensor.adoc [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+===== Methods
+
+[source, java]
+----
+include::../../../../../../pi4micronaut-utils/src/main/java/com/opensourcewithslu/inputDevices/UltraSonicSensorHelper.java[tags=method]
+----
+
+===== An Example Controller
+
+====== This controller uses the push button to turn an LED on and off
```
Comment (ruthvikm) [inline pi4micronaut-utils/src/docs/asciidoc/components/inputComponents/ultraSonicSensor.adoc:73]: This is not related to ultrasonic

**Review by ruthvikm — CHANGES_REQUESTED** [review]

**Review by ruthvikm — COMMENTED** [review]
No images are uploaded. Make sure you include the images in img folder

**Review by ruthvikm — APPROVED** [review]
